libraries/engage/components/SheetList/components/Item/index.jsx

```diff
@@ -52,16 +52,12 @@ class Item extends Component {
    */
   renderContent() {
     const {
-      isDisabled, isSelected, title, image, rightComponent,
+      isDisabled, title, image, rightComponent,
     } = this.props;
 
-    let gridStyles = styles.grid;
+    const gridStyles = styles.grid;
```

Comment:
Moving the `selected` class causes, that clicking on a selected item comes highly visible, since the inner section of the item changes it color. 

![Bildschirmfoto 2019-06-24 um 09 12 40](https://user-images.githubusercontent.com/13087792/59999662-cfdf7380-9662-11e9-8129-65e4c50c4324.png)
 
On GMD there is now too much space now when options have additional prices.
![Bildschirmfoto 2019-06-24 um 09 47 03](https://user-images.githubusercontent.com/13087792/60000792-35346400-9665-11e9-835a-b01c2065d329.png)